src/sidebar/components/UserMenu.tsx

```diff
@@ -110,6 +114,11 @@ function UserMenu({ frameSync, onLogout, settings }: UserMenuProps) {
           )}
           <MenuItem label="Open notebook" onClick={() => onSelectNotebook()} />
         </MenuSection>
+        {settings.dashboard?.showEntryPoint && (
+          <MenuSection>
+            <MenuItem label="Show dashboard" onClick={() => dashboard.open()} />
```

Comment:
Perhaps "Open dashboard" would be better since I believe the plan is to open a new tab? In future we could possibly simplify this to just "Dashboard", although we'd need to update other menu items for consistency.